Pages/UITasks.js

```diff
@@ -0,0 +1,75 @@
+var homePage = require('../Pages/homePage.js'),
+    loginPage = require('../Pages/loginPage'),
```

Comment:
please create separated folder "Tasks" for UITasks.js class

Reply:
I created folder and moved UITasks.js class to it.

Reply:
I created separated folder for UITasks.js class